dep/jsoncpp/README.md

```diff
@@ -2,6 +2,15 @@
 
 [Amalgamated](https://github.com/open-source-parsers/jsoncpp/wiki/Amalgamated)
 from source commit
-[ddabf50](https://github.com/open-source-parsers/jsoncpp/commit/ddabf50f72cf369bf652a95c4d9fe31a1865a781),
-release 1.8.4.
+[6aba23f](https://github.com/open-source-parsers/jsoncpp/commit/6aba23f4a8628d599a9ef7fa4811c4ff6e4070e2),

```

Comment:
Huh, weird that it thinks this is still in the diff....

Reply:
hmmmmmm. you may want to checkout master and cherry-pick just the one commit, then force push